generators/server/templates/gradle/_yeoman.gradle

```diff
@@ -1,12 +1,20 @@
-import org.apache.tools.ant.taskdefs.condition.Os
-apply plugin: 'com.moowork.gulp'
+apply plugin: 'com.moowork.node'
 
-task bower(type: Exec) {
-    if (Os.isFamily(Os.FAMILY_WINDOWS)) {
-        commandLine 'cmd', '/c', 'bower', 'install'
-    }else{
-        commandLine 'bower', 'install'
-    }
+task bower(type: NodeTask) {
+    description = "Installs dependencies using Bower"
+    script = file("${project.projectDir}/node_modules/bower/bin/bower")
```

Comment:
Yes. On Linux node_modules/.bin/webpack is a symlink to node_modules/webpack/bin/webpack.js. So IMO this platform check was useless and both could use node_modules/webpack/bin/webpack.js directly.